app/filtration/filtrationPage.tsx

```diff
@@ -71,14 +119,51 @@ export default function FiltrationPage(): React.JSX.Element {
       <ScrollView
         style={styles.scrollView}
         contentContainerStyle={styles.contentContainer}
-        showsVerticalScrollIndicator={true}>
+        showsVerticalScrollIndicator={true}
+        scrollEnabled={scrollEnabled}>
         <View style={styles.topSpacer} />
 
         <View style={styles.filtrationsContainer}>
-          <View style={styles.filtrationContainer}>
-            <Text style={styles.filtrationTitle}>ToDo</Text>
-            <Text style={styles.filtrationSubtitle}>ToDo</Text>
-          </View>
+          <SexSelectorOtter
+            title="Płeć"
+            subtitle="Jakiej płci szukasz?"
+            value={selectedSex}
+            onChange={(newSex) => setSelectedSex(newSex)}
+            multiSelect={true}
+          />
+          <SliderOtter
+            title="Maksymalny Dystans"
+            subtitle="Do jakiej odległości wydra może odpłynąć?"
+            value={distanceRange}
+            onChange={(newDistance) => setDistanceRange(newDistance as number)}
+            minValue={1}
+            maxValue={100}
+            step={1}
+            rangeBetween={false}
+            onSlidingStart={() => setScrollEnabled(false)}
+            onSlidingComplete={() => setScrollEnabled(true)}
+          />
+          <SliderOtter
+            title="Wiek"
+            subtitle="Wybierz zakres wieku"
+            value={ageRange}
+            onChange={(newRange) => setAgeRange(newRange as [number, number])}
+            minValue={18}
+            maxValue={100}
+            step={1}
+            rangeBetween={true}
+            onSlidingStart={() => setScrollEnabled(false)}
+            onSlidingComplete={() => setScrollEnabled(true)}
+          />
+          <SearchingSelectorOtter
+            title="Szukam"
+            subtitle="Czego szukasz?"
+            value={selectedSearching}
```

Comment:
Tutaj bym troche zmienił nazewnictwo, tak żeby odróżnić to od tego co ustawiamy na swoim profilu
```suggestion
title="Poszukiwania drugiej wyderki"
subtitle="Filtruj czego szuka druga wyderka"
```

Reply:
Preview:
![image](https://github.com/user-attachments/assets/edceadda-2600-454b-ab29-e729cb5071b8)

Reply:
Okej, brzmi spoko. Ogólnie jest pewna zasada w takich nazewnictwach by jak najbardziej ukrócać w takich przypadkach. Czyli na przykład "Poszukiwania wyderki" albo "Poszukiwania wyderek" gdzie słowo "drugiej" jest takim zapełniaczem a zabiera miejsca horyzontalnie. Tak samo w opisie słowo "filtruj" też bym zastąpił bo użytkownik jest już w zakładce z filtracjami. Mało istotne zasady ale tylko informacyjnie xD Jak na moje może być i jest spoko.